### DISCUSSION TITLE
有没有成功使用Warp节点的大哥？

### DISCUSSION THREAD

**Discussion by Travel2Here** [Openwrt-Passwall/openwrt-passwall2#287] [General]
极客们近来流行薅warp服务的羊毛，但提取出的配置，在pw里用wireguard协议建节点并不能用，求成功人士分享

![image](https://github.com/xiaorouji/openwrt-passwall2/assets/99887507/d50b169a-0e4a-4b81-bd9d-38b30fb989f7)

![image](https://github.com/xiaorouji/openwrt-passwall2/assets/99887507/4cd5340d-5e3e-4d2d-9f74-939a807c8467)

**Comment by etsam** [comment]
我是在虚拟机里面搞了个LINUX然后在上面跑WARP ，在装个X-UI做个vless给PW2使用，用是可以用，就是速度时快时慢 放弃了

**Comment by handsomeji** [comment]
需要節點支持UDP，並且使用sing-box。
默認節點使用wg節點，並使用前置代理。

**Comment by Travel2Here** [comment]
> 需要節點支持UDP，並且使用sing-box。 默認節點使用wg節點，並使用前置代理。

算了，这个是半年前的疑问，早放弃了。这羊毛咱不薅了。（速度不行也不稳）
